internal/setup/setup.go

```diff
@@ -242,9 +242,14 @@ func order(pkgs map[string]*Package, keys []SliceKey) ([]SliceKey, error) {
 	return order, nil
 }
 
-var fnameExp = regexp.MustCompile(`^([a-z0-9](?:-?[.a-z0-9+]){2,})\.yaml$`)
+// fnameExp matches the slice definition file basename.
+var fnameExp = regexp.MustCompile(`^([a-z0-9](?:-?[.a-z0-9+]){1,})\.yaml$`)
```

Comment:
Maybe this is too much for a bugfix but I feel a little uneasy by changing a regexp and not having tests for that particular expression. I see we are testing the expression as part of `kname` but they are not "linked" in code. What do you think of?
```go
var pkgNameExpStr = "([a-z0-9](?:-?[.a-z0-9+]){1,})"
var sliceNameExpStr = "([a-z](?:-?[a-z0-9]){2,})"

var fnameExp = regexp.MustCompile(`^` + pkgNameExpStr + `\.yaml$`)
var knameExp = regexp.MustCompile(`^` + pkgNameExpStr + `_` + sliceNameExpStr + `$`)
```

The tests could then be created for pkg and slice and combined programmatically for kname.

Reply:
Huh, good point. I will change it accordingly.